client/packages/host/src/api/hooks/settings/namePropertyData.ts

```diff
@@ -69,7 +69,7 @@ export const gapsNameProperties: {
       key: 'gas_availability',
       name: 'Gas Availability',
       valueType: PropertyNodeValueType.String,
-      allowedValues: 'Not availability,Available,Irregular,Unknown',
+      allowedValues: 'No availability,Available,Irregular,Unknown',
```

Comment:
btw this was fix for [#4231](https://github.com/msupply-foundation/open-msupply/issues/4231)